### ISSUE TITLE
Question about torch version

### ISSUE THREAD

**Issue by xiaobaiaaaaa** [huanglianghua/siamfc-pytorch#59]
When I run train.py, the following error occurs：

UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)

Because I am a 40-series graphics card (cuda=11.3), I heard that it is enough to reduce torch to 1.0.1, but cuda is not enough, how to solve this problem?
Thanks
